src/platform/forms-system/src/js/review/SubmitController.jsx

```diff
@@ -45,8 +45,9 @@ class SubmitController extends Component {
   };
 
   handleSubmit = () => {
-    const { form, formConfig, pageList, trackingPrefix } = this.props;
+    const { form, formConfig, pageList, trackingPrefix, user } = this.props;
     const { formId, data, submission } = form;
+    const isLoggedIn = user?.login?.currentlyLoggedIn;
```

Comment:
There's a selector for this in `src/platform/user/selectors.js` that can be used in the `mapStateToProps`.